wsgiservice/resource.py

```diff
@@ -457,13 +500,12 @@ def convert_param(self, method, param, value):
                 value is invalid for the given method and parameter.
         """
         rules = self._get_validation(method, param)
-        if not rules or not rules.get('convert'):
+        if not rules or not rules.get('convert') or (value is None and not rules['mandatory']):
```

Comment:
I don't know the architecture of this, but it interesting that we do this check *inside* check_param. I would have expected to not call the method at all for none and not mandatory values. More logic here makes this harder to test.